### PR TITLE
:sparkles: Use the posts submodule (renamed content) as a collections_dir

### LINKED ISSUE #17: Make images a collection, and group them with posts as a submodule using collections_dir
cf https://stackoverflow.com/a/49428493/

### PR DESCRIPTION
**Pull request by moeenio**
goes along with the latest changes in csourcesc/posts

fixes #17

### CODE REVIEW

File: 404.md [hunk 1]
```diff
@@ -4,6 +4,6 @@ permalink: "/404.html"
 layout: page
 ---
 
-![Lukas the bear is lost...]({{ site.baseurl }}/images/404.svg)
+![Lukas the bear is lost...]({{ site.baseurl }}/404.svg)
```
Comment (ghost) [inline 404.md:7]: His name is Lukas? Didn't know lol

**Review by ghost — APPROVED** [review]

File: 404.md (hunk 1 above)
Comment (moeenio) [reply 404.md:7]: This dates back to the old CatSource. Fenks for fix

Comment (ghost) [reply 404.md:7]: Sorry lol

**Comment by ghost** [comment]
Able to merge?

File: 404.md (hunk 1 above)
Comment (moeenio) [reply 404.md:7]: wot

**Comment by moeenio** [comment]
no

**Comment by ghost** [comment]
`normal title`: unable to merge  
`emoji + title`: ok go merge now

**Comment by moeenio** [comment]
No, just that not using the PRs title when squash merging auto-changes the PRs title

**Comment by ghost** [comment]
oh ok

**Pull request merged**